runner/java_parser/src/main.rs

```diff
@@ -8,16 +8,16 @@ extern crate cfgrammar;
 use lrpar::{ParseRepair, LexParseError, Lexer};
 
 // Using `lrlex_mod!` brings the lexer for `calc.l` into scope.
-lrlex_mod!(java7_l);
-// Using `lrpar_mod!` brings the lexer for `calc.l` into scope.
-lrpar_mod!(java7_y);
+lrlex_mod!("java7.l");
+// Using `lrpar_mod!` brings the parser for `calc.l` into scope.
```

Comment:
/calc.l/calc.y

Reply:
Oops. Fixed in https://github.com/softdevteam/error_recovery_experiment/pull/27/commits/2a0e29bdc420bcb5d899a5d7e471ec06d7c98c38